libs/ui/elements/src/lib/markdown-editor/markdown-editor.component.ts

```diff
@@ -0,0 +1,52 @@
+import { CommonModule } from '@angular/common'
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core'
+import { FormsModule } from '@angular/forms'
+import { MarkdownParserComponent } from '../markdown-parser/markdown-parser.component'
+import { TranslateModule } from '@ngx-translate/core'
+import { ButtonComponent, TextAreaComponent } from '@geonetwork-ui/ui/inputs'
+import { MatIconModule } from '@angular/material/icon'
+import { MatTooltipModule } from '@angular/material/tooltip'
+
+@Component({
+  selector: 'gn-ui-markdown-editor',
+  templateUrl: './markdown-editor.component.html',
+  styleUrls: ['./markdown-editor.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [
+    CommonModule,
+    FormsModule,
+    MatIconModule,
+    MatTooltipModule,
+    ButtonComponent,
+    TextAreaComponent,
+    MarkdownParserComponent,
+    TranslateModule,
+  ],
+})
+export class MarkdownEditorComponent {
+  @Input() label: string
+  @Input() tooltip: string
+  @Input() helperText: string
```

Comment:
marking these as optional might make things more clear, even though it doesn't change anything for typechecking